docs/LDES.md

```diff
@@ -0,0 +1,111 @@
+
+- an LDES as discussed earlier. We can use the caching headers here to clarify that the resource isn't expected to change for another month or so. We can also terminate an LDES when we know no more data is going to change, e.g. for election results.
+- we can make an interface (API, not LDES) where they just fetch the latest version.
+
+## LDES setup
+
+The LDES spec can be found [here](https://semiceu.github.io/LinkedDataEventStreams/).
+We will be using the implementation provided by [redpencil.io on their github](https://github.com/redpencilio/fragmentation-producer-service) to publish the LDES. It will be fed using our own service that monitors deltas on our SEAS instance.
+
+We'll use a time-fragmenter, one stream will be set up per type of instance that we will share, with pagination and the following setup for versioning:
+
+    @prefix dct: <http://purl.org/dc/terms/>.
+    @prefix ext: <http://mu.semte.ch/vocabularies/ext/>.
+    @prefix xsd: <http://www.w3.org/2001/XMLSchema#>.
+    @prefix ldes: <https://w3id.org/ldes#>.
+    @prefix tree: <https://w3id.org/tree#>.
+    @prefix prov: <http://www.w3.org/ns/prov#>.
+
+    <stream> ldes:timestampPath prov:generatedAtTime ;
+             ldes:versionOfPath dct:isVersionOf .
+             ldes:retentionPolicy ext:retention .
+
+    ext:retention a ldes:DurationAgoPolicy ;
+          tree:value "P1Y"^^xsd:duration .
+
+This also means that we promise that we will keep the data for at least one year after it was created.
+The url of the type of entity will be based on the resources path of the entity type in
+
+## Links to Entities
+
+When other entities are referenced from an LDES feed instance, the original URI of that entity is used, not the versioned URI.
+
+For instance, if this snippet is part of a `mandaat:Mandataris` instance that refers to a `person:Person`, its `mandaat:isBestuurlijkeAliasVan` predicate refers to the true URI of the person, not the person's versioned URI.
+
+    @prefix dct: <http://purl.org/dc/terms/>.
+    @prefix ext: <http://mu.semte.ch/vocabularies/ext/>.
+    @prefix xsd: <http://www.w3.org/2001/XMLSchema#>.
+    @prefix mandaat: <http://data.vlaanderen.be/ns/mandaat#>.
+
+    ext:mandatarisVersioned1 dct:isVersionOf ext:mandatarisTrueUri
+     dct:issued "2024-02-14T14:05:00.000Z"^^xsd:dateTime;
+     mandaat:isBestuurlijkeAliasVan ext:truePersonUri.
+
+    ext:personVersioned dct:isVersionOf ext:truePersonUri.
+     dct:issued "2024-02-14T14:05:00.000Z"^^xsd:dateTime;
+     foaf:name "Jos" .
+
+Sidenote: it's also not a given that a versioned person URI exists in our LDES. They are only created if it's a person that wasn't in the election results and we had to create ourselves (e.g. for the head of an OCWM being made schepen).
```

Comment:
This versioned URI of a person is a bit confusing here because of the description above about versioned entities of mandatarissen. Would be good to clarify here that this is something different and is specific to how LDES works.